torch_xla/csrc/tensor.cpp

```diff
@@ -789,10 +789,12 @@ c10::SymNode XLASymNodeImpl::is_channels_last_strides_3d(
   XLA_CHECK(false) << "XLASymNodeImpl::" << __FUNCTION__
                    << " has not been implemented.";
 }
+
+// xw32: what's the reason that this function should never be called in practice?
 c10::SymNode XLASymNodeImpl::is_non_overlapping_and_dense(
```

Comment:
hi @ezyang , what's the reason that both `is_non_overlapping_and_dense` and `sym_or` won't be used in practice respectively so that we can just return an `ErrorNode`?

Reply:
it's because I happen to know that these are only used to compute contiguity fields on tensors like "is non overlapping and dense" and if they are never fetched it is fine for them to error only if poked